intellij/src/saros/intellij/ui/tree/ContactTreeRootNode.java

```diff
@@ -22,14 +25,18 @@
  *
  * <p>It registers as {@link IRosterListener} and keeps the displayed contacts in sync with the
  * server.
+ *
+ * <p><b>NOTE:</b>This component and any component added here must be correctly torn down when the
```

Comment:
Maybe it would be more helpful to put such information to a central place (documentation etc.).

Reply:
Yeah, that would probably be a good idea. We currently don't have a section where IDE specific implementation details/restrictions can be listed. Will look into adding something like that.

Reply:
I did not know where to put the section. We need a better separation for IDE specific topics. I created an issue for the separation issue (#593) and for this missing documentation (#594).